src/main/resources/env/qa.properties

```diff
@@ -1,5 +1,6 @@
 baseUri=http://3.65.154.68
 port=8089
 basePath=api
+baseUrl=http://3.65.154.68:8089/batteriesoverview
```

Comment:
I suggest changing the name of this property key into something more suggestive (like batteriesOverviewURI)